legacy/index.tsx

```diff
@@ -1,12 +1,12 @@
 import ReactDOM from "react-dom/client";
```

Comment:
I now realize that no longer having `@mdn/yari` in our dependencies means this transitive dependency is no longer installed.

In theory, we could use `vendor/yari` as a local dependency, but that would break Dependabot updates.